src/components/overview/unresolvedTicketsBox/helper.ts

```diff
@@ -0,0 +1,108 @@
+export interface IUnresolvedTickets {
```

Comment:
IUnresolvedTicket - you always describe  a singular

Reply:
fix: IUnresolvedTicket